ui/test-file/summary/index.tsx

```diff
@@ -148,6 +152,9 @@ export default function FileSummary({
           <Info color="primary">
             <Code size={14} /> <InfoLabel>{testCount} Tests</InfoLabel>
           </Info>
+          <Info color="secondary">
+          <Circle size={14} />{" "} <InfoLabel>{todoCount} Todos</InfoLabel>
```

Comment:
Also is this space necessary?

Reply:
Don't think so, just copied from the other example as is

Reply:
deleted on commit ca7f016